contrib/!lang/erlang-elixir/README.md

```diff
@@ -126,5 +160,12 @@ Note that `EDTS` is disable on Windows.
 <kbd>SPC m x b</kbd>  | Run the current buffer through `elixir`
 <kbd>SPC m x f</kbd>  | Run `elixir` with the given filename
 
+##### Code Definition Jump
+
+    Key Binding     |                 Description
+--------------------|------------------------------------------------------------
+<kbd>SPC m .</kbd>  | Jump to the elixir expression definition at point.
+<kbd>SPC m ,</kbd>  | Pop back to where <kbd>SPC m .</kbd> was last invoked.
+
```

Comment:
Add `SPC m g g` to go to definition under point. For now we don't have any conventions to pop back to the previous location though.